### DISCUSSION TITLE
Antivirus reports malware in 2024.10.01 update.

### DISCUSSION THREAD

**Discussion by Darmorque** [joncampbell123/dosbox-x#5293] [Q&A]
Hi,

I have Dosbox-X 2023.10.06 installed through winget. Now I'm trying to update it to the 2024.10.01 version, but no matter what, my antivirus software reports HEU_AEGIS_HttpDownloadChecker threat present in installer package. I suppose it might be some sort of false flag, but you never know and it's always better to ask someone smarter than me, what might be the case and is it safe to install anyway?

**Comment by joncampbell123** [comment]
Where did you get the installer package?

If this came from the dosbox-x website or GitHub that would be very concerning. Builds here are built using a CI workflow.

**Reply by Darmorque** [reply]
As I wrote above, I had the package installed through Winget, and I'm trying to update it the same way. I use Trend Micro antivirus and I found some not-that-clear info about it detecting HEU_AEGIS_HttpDownloadChecker meaning it's just hypersensitive to some things, but it's better to get some sort of confirmation of it being safe anyway.

**Comment by maron2000** [comment]
It's only up to you to think HEU_AEGIS_HttpDownloadChecker  is false positive or not.
Otherwise, check the source code and build it yourself.
That's one of the benefits of open source software.

One suggestion is to consult virustotal, but this doesn't guarantee anything.
https://www.virustotal.com/gui/url/ccc9a90cbfcff86bf3dc8890e81d57a59c28e89cc5e61a0fe990da27082eaf95
